tfdf-penguins/index.html

```diff
@@ -0,0 +1,165 @@
+<!-- Copyright 2023 Google LLC. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+============================================================================== -->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Yggdrasil Decision Forests in TensorFlow.JS</title>
+
+  <!-- Import @tensorflow/tfjs -->
+  <!-- <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script> -->
+  <script src="./node_modules/@tensorflow/tfjs/dist/tf.js"></script>
```

Comment:
TODO(mattSoulanille): Replace this with a CDN url after tfjs 4.3.0 is published.